setup.py

```diff
@@ -60,6 +60,7 @@
         'simplejson',
         'spectra',
         'networkx==1.*',
+        'enum34',
```

Comment:
Hey! Quick question - what is this and is it needed? ;) I'm always a little paranoid about adding dependencies (especially after the trouble we're having with a couple at the moment). Feel free to grab me in person later..

Phil